models/org_team.go

```diff
@@ -37,6 +39,27 @@ type Team struct {
 	CanCreateOrgRepo        bool        `xorm:"NOT NULL DEFAULT false"`
 }
 
+// AfterLoad is invoked from XORM after setting the values of all fields of this object.
+func (t *Team) AfterLoad() {
```

Comment:
It's better to use `AfterLoad(sess *xorm.Session)` and resue `see` to load organization.